sdk/helper/ldaputil/config.go

```diff
@@ -559,3 +561,51 @@ func (c *ConfigEntry) Validate() error {
 	}
 	return nil
 }
+
+func ConvertConfig(cfg *ConfigEntry) *capldap.ClientConfig {
+	// cap/ldap doesn't have a notion of connection_timeout, and uses a single timeout value for
+	// both the net.Dialer and ldap connection timeout.
+	// So take the smaller of the two values and use that as the timeout value.
+	minTimeout := min(cfg.ConnectionTimeout, cfg.RequestTimeout)
+	urls := strings.Split(cfg.Url, ",")
+	config := &capldap.ClientConfig{
```

Comment:
Does cap/ldap support our [max_page_size](https://developer.hashicorp.com/vault/api-docs/auth/ldap#max_page_size) for group queries?

Reply:
Hmm, it doesn't appear so. @jimlambrt can you confirm?

Reply:
Not yet.  It's def something we can add.  I just overlooked it.  Boundary's going to need this as well.

Reply:
This has been added to cap/ldap!